src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/CalendarRepositoryTest.kt

```diff
@@ -133,6 +133,57 @@ class CalendarRepositoryTest : IntegrationTestBase() {
     assertThat(result[expectedBedKey]).isEmpty()
   }
 
+  @Test
+  fun `Results are correct for a Premises with a Room & Bed and a non-arrived lost bed and booking`() {
```

Comment:
Should this read:

`Results are correct for a Premises with a Room & Bed and a lost bed and non-arrived booking`